proxy/src/bind.rs

```diff
@@ -60,7 +63,13 @@ pub enum Host {
     NoAuthority,
 }
 
-pub type Service<B> = Reconnect<NewHttp<B>>;
+/// Reconstruct HTTP/1.x URIs.
+#[derive(Copy, Clone, Debug)]
+pub struct ReconstructUri<S> {
+    upstream: S
```

Comment:
strong preference to avoid the word `upstream.` we tend to use `inner`

Reply:
I thought I've seen fairly mixed usage, but I'm happy to change it. Might be nice to have preferred naming schemes written down somewhere?